docs/integrations/saas-cloud/box.md

```diff
@@ -80,153 +80,326 @@ _sourceCategory=box  type "event_type" login
-     ```
-     curl -X GET 'the above url'
-     ```
-    * **For Windows**, open a Powershell window and run:
-    ```
-    Invoke-WebRequest 'the above url' -Method Get
-    ```
-
-  If everything was successful, you should see the message “`Thank you for granting access for SumoLogic BoxCollector”` somewhere in the return value. If you see an error regarding an expired authorization code instead, make sure you finish this step within 30 seconds of the previous step as noted above.
-
-8. Once permissions are granted, the script saves the access token to a local file—the default location is `${path}/data` or `./data`. Verify that the file was created. If not, repeat the authentication steps.
-
-  **On some Windows machines, the SumoJanus folder has “Read only” permission by default. Make sure you allow Write permission.**
-
-  The path to the token file is configured in the `conf/sumologic.properties` file, under the property `token_path`.
-
-9. (Optional) Test the script manually by going to the **sumojanus-box **folder and running one of the following commands.
-   * For Linux systems, run this command:
-    ```bash
-    bin/SumoJanus_Box.bash
-    ```
-   * **For Windows** systems, run this command:
-    ```bash
-    bin\SumoJanus_Box.bat
-    ```
-
-  You should now see a list of results of collected Box events.
-
-10. Close the CLI (Windows) or shell (Linux) window to kill the running script. By default it runs for 30 minutes.
-
-
-### Step 6: Configure a Source
-
-For guidance creating your source category naming convention, see [Best Practices: Good Source Category, Bad Source Category](/docs/send-data/best-practices).
-
-To configure a source, do the following:
-
-1. Configure a [Script Source](/docs/send-data/installed-collectors/sources/Script-Source). Collectors using version 19.245-4 and later do not allow Script Sources to run by default.
-To enable Script Sources you must set the Collector parameter `enableScriptSource` in [user.properties](/docs/send-data/Installed-collectors/collector-installation-reference/user-properties) to true and [restart](/docs/send-data/collection/start-stop-collector-using-scripts.md) the Collector.
-2. Configure the source fields:
-    1. **Name**. (Required) BoxCollector. (Description is optional.)
-    2. **Source Category**. (Required) box
-    3. **Frequency** (Required) **Every 5 Minutes**
-    4. **Specify a timeout for your command**: Active the checkbox and select **60 Minutes**
-    5. **Command** (Required)`/bin/bash` (specify the correct path on your system)
-    6. **Script** (Required) Use the path to `sumojanus`, such as: `/home/ubuntu/sumojanus-box/bin/SumoJanus_Box.bash`. Do not select “Type the script to execute.”
-    7. **Working Directory.** `/home/ubuntu/sumojanus-box`.
-3. Click **Save**.
+### Authentication
+
+You'll need a Box App Key, App Secret, and Access Code to provide to Sumo Logic.
+
+The following steps must be followed to get the credentials:
```

Comment:
Complete the following steps to get the credentials:

Reply:
done